src/main/java/com/williamfeliciano/sba_movies/dtos/CreateMovieDto.java

```diff
@@ -0,0 +1,45 @@
+package com.williamfeliciano.sba_movies.dtos;
+
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+public class CreateMovieDto {
```

Comment:
What's the difference between this DTO and the `UpdateMovieDto`? Is there any difference with `MovieDetailsDto`? 

Why don't have only a single DTO? This way, when adding or renaming a field, you don't need to edit 3 DTOs